app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
```

Comment:
- [ ] [Optional][Highly recommended] As mentioned in the [devise tutorial](https://github.com/heartcombo/devise#controller-filters-and-helpers) Please use this helper method `before_action :authenticate_user!` to protect the data of the users so the user needs to log in first to see this data. You can use it here in the application controller to protect all controllers.